architecture/additional_concepts/sdn.adoc

```diff
@@ -65,6 +65,14 @@ given node is allocated 510 addresses that it can assign to the containers
 running on it. The size and address range of the cluster network are
 configurable, as is the host subnet size.
 
+[NOTE]
+====
+If a subnet extends into the next higher octet, we rotate the bits of the subnet number so subnets
```

Comment:
We should probably say that this is a behavior of the openshift SDN plugins (as the next paragraph says)

Also, we should probably give the rationale for this (i.e. we do it so that the SDN ip addresses for the nodes are all the .1 addresses to make it easier for humans to follow).